winpkg/ddev.nsi

```diff
@@ -616,10 +616,10 @@ SectionEnd
  */
 LangString DESC_SecDDEV ${LANG_ENGLISH} "Install ${PRODUCT_NAME_FULL} (required)"
 LangString DESC_SecAddToPath ${LANG_ENGLISH} "Add the ${PRODUCT_NAME} (and sudo) directory to the global PATH"
-LangString DESC_SecSudo ${LANG_ENGLISH} "Sudo for Windows (github.com/mattn/sudo) allows for elevated privileges which are used to add hostnames to the Windows hosts file (required)"
+LangString DESC_SecSudo ${LANG_ENGLISH} "Sudo for Windows (github.com/ddev/gsudo) allows for elevated privileges which are used to add hostnames to the Windows hosts file (required)"
```

Comment:
Sorry, I think we're always using https://github.com/gerardog/gsudo - I think that's the right reference here.

But per https://github.com/ddev/gsudo/releases/tag/v0.7.3, the reason we're bundling an old one here is because he doesn't publish a raw binary. So we'd have to unbundle it. I guess.  I requested this in https://github.com/gerardog/gsudo/issues/88, he wasn't willing.

Reply:
Is it obvious to you how we could download and unbundle the binary during the build process? We could certainly do it in the Makefile.

Reply:
Is it obvious to you how we could download and unbundle the binary during the build process? We could certainly do it in the Makefile.

Reply:
Is it obvious to you how we could download and unbundle the binary during the build process? We could certainly do it in the Makefile.